modules/process/stacktraces_processlist.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package process
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+)
+
+// WriteProcesses writes out processes to a provided writer
+func WriteProcesses(out io.Writer, processes []*Process, processCount int, goroutineCount int64, indent string, flat bool) error {
+	if goroutineCount > 0 {
+		if _, err := fmt.Fprintf(out, "%sTotal Number of Goroutines: %d\n", indent, goroutineCount); err != nil {
+			return err
+		}
+	}
+	if _, err := fmt.Fprintf(out, "%sTotal Number of Processes: %d\n", indent, processCount); err != nil {
+		return err
+	}
+	if len(processes) > 0 {
+		if err := WriteProcess(out, processes[0], "  ", flat); err != nil {
```

Comment:
I don't think this is correct, shouldn't it be
```suggestion
		if err := WriteProcess(out, processes[0], indent+"  ", flat); err != nil {
```
?

Reply:
This code is unchanged from previously.

Adding the indent here would cause things to become too indented. It produces clearer output this way. 

Have you tried your proposed change with `gitea manager processes`  +- `--stacktraces`?

Reply:
I believe you that it produces worse results, it simply appeared to be missing based on the code above it.
What about
```suggestion
		if err := WriteProcess(out, processes[0], /*Ignore indent to keep the result readable*/"  ", flat); err != nil {
```
or something like that?